language-service/language_service/service/vocab/__init__.py

```diff
@@ -0,0 +1,34 @@
+"""
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First word of the docstring should not be `This`](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5112709)